R/create_protocol.R

```diff
@@ -266,8 +267,8 @@ create_sfp <- function(
 #' Rmarkdown files listed underneath the src folder.
 #' The search will be restricted to files of a given protocol type.
 #'
-#' @param protocol_type A character string equal to sfp (default), sip, sap or
-#' sop.
+#' @param protocol_type A character string equal to sfp (default), sip, sap,
+#' sop or spp.
```

Comment:
I'm not sure what was discussed before about `protocol_type`, but this is function `create_sfp()`, so either it is for type `sfp` only or the function's name is too narrow. Is the argument still used? The templates are protocol_type-specific I guess.

Reply:
I see I'm mistaken here, going to delete this. :roll_eyes: